cypress/integration/realm_settings_client_policies_test.spec.ts

```diff
@@ -1,35 +1,32 @@
 import SidebarPage from "../support/pages/admin_console/SidebarPage";
 import LoginPage from "../support/pages/LoginPage";
 import RealmSettingsPage from "../support/pages/admin_console/manage/realm_settings/RealmSettingsPage";
-import {
-  keycloakBefore,
-  keycloakBeforeEach,
-} from "../support/util/keycloak_hooks";
+import { keycloakBefore } from "../support/util/keycloak_hooks";
 import AdminClient from "../support/util/AdminClient";
+import ModalUtils from "../support/util/ModalUtils";
 
 const loginPage = new LoginPage();
 const sidebarPage = new SidebarPage();
+const realmSettingsPage = new RealmSettingsPage();
+const modalUtils = new ModalUtils();
 
 describe("Realm settings client policies tab tests", () => {
   const realmName = "Realm_" + (Math.random() + 1).toString(36).substring(7);
-  const realmSettingsPage = new RealmSettingsPage(realmName);
 
   beforeEach(() => {
-    keycloakBeforeEach();
-    sidebarPage.goToRealm(realmName);
+    keycloakBefore();
+    loginPage.logIn();
     sidebarPage.goToRealmSettings();
     cy.findByTestId("rs-clientPolicies-tab").click();
     cy.findByTestId("rs-policies-clientPolicies-tab").click();
   });
 
-  before(() => {
-    keycloakBefore();
-    new AdminClient().createRealm(realmName);
-    loginPage.logIn();
+  before(async () => {
+    await new AdminClient().createRealm(realmName);
   });
 
-  after(() => {
-    new AdminClient().deleteRealm(realmName);
+  after(async () => {
+    await new AdminClient().deleteRealm(realmName);
   });
```

Comment:
No need for async syntax if the returned value is a Promise.
```suggestion
  after(() => new AdminClient().deleteRealm(realmName));
```

Reply:
This is from the rebase that got changed back to what it was before. I'll revert it